force-app/main/default/lwc/psElevateTokenHandler/psElevateTokenHandler.js

```diff
@@ -56,6 +56,9 @@ class psElevateTokenHandler {
         const sandboxEnhancedUrl =  `https://${domainInfo.orgDomain}--${namespace}.sandbox.vf.force.com`;
         const sandboxEnhancedLightning =  `https://${domainInfo.orgDomain}--${namespace}.sandbox.lightning.force.com`;
         const sandboxEnhancedCanon =  `https://${domainInfo.orgDomain}--${namespace}.sandbox.my.salesforce.com`;
+        const scratchEnhancedUrl =  `https://${domainInfo.orgDomain}--${namespace}.scratch.vf.force.com`;
```

Comment:
I think we can remove this change since @balsamhindi has it covered in  #7131

Reply:
Sure, I needed it for my testing, but didn't need to commit it.